src/hugr/validate.rs

```diff
@@ -95,6 +89,7 @@ impl<'a> ValidationContext<'a> {
         let flags = optype.validity_flags();
 
         // The Hugr can have only one root node.
+        // The root node has no edges.
```

Comment:
Perhaps we should check. An edge to a child would be caught on the child's validation,
but a loop edge on the root node would not be checked.